### PR TITLE
veteran box remote and toggle sorting

### PR DESCRIPTION
**Pull request by aryehbeitz**
### On the home page where you search for a host:

- I removed the veteran checkbox 

- I added support for clicking on a column header which changes sort direction - toggles on each click.

I also did the changes for the manager page, but I didn't test

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -12,7 +12,9 @@ def home
   	}
 
     @hosts = sort_by_field(@hosts, params[:sort] || 'user.full_name')
-
+    if params[:reverse_ordering].to_i == 0
+      @hosts = @hosts.reverse
```
Comment (gaaady) [inline app/controllers/pages_controller.rb]: why do you need to reverse here in not in sort_by_field?

**Comment by gaaady** [comment]
@uriklar please give @aryehbeitz write permissions
or give me admin permissions

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (aryehbeitz) [inline app/controllers/pages_controller.rb]: if i applied it in the sort_by_field, it would have to be applied to each header, with a conditional on each.

**Comment by aryehbeitz** [comment]
[חיפוש סלון: להוריד v ״סלון עם וטרן״](https://trello.com/c/XcsOvf6e/4-v)

**Pull request merged**